uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Importer/LeavePeriodEntitlementTest.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveBalanceChange as LeaveBalanceChange;
+use CRM_HRLeaveAndAbsences_Test_Fabricator_AbsencePeriod as AbsencePeriodFabricator;
+use CRM_HRLeaveAndAbsences_Test_Fabricator_AbsenceType as AbsenceTypeFabricator;
+
+/**
+ * Class CRM_HRSampleData_Importer_LeavePeriodEntitlementTest
+ *
+ * @group headless
+ */
+class CRM_HRSampleData_Importer_LeavePeriodEntitlementTest extends CRM_HRSampleData_BaseCSVProcessorTest {
+
```

Comment:
Please remove empty line here.

Reply:
Why? Is there any code style rule about that?

Reply:
oops, actually thought the line was after a function definition.